setup/jest.config.js

```diff
@@ -274,8 +274,6 @@ module.exports = {
     'app/src/main.js',
     'src/modules/account/hooks/useDecryptionAccount.js',
     'src/redux/selectors.js',
-    'src/redux/selectors.js',
```

Comment:
NRTTL
When i receive a socket message regarding a transaction it make a lot of additional network requests. This results in some widgets to show loading state for a while. Is it intentional that we shall make these network requests when receiving a socket transaction?
<img width="627" alt="image" src="https://user-images.githubusercontent.com/8784876/205983869-89ed751f-ddb9-43ca-b73c-622f1cf23854.png">

Reply:
@oskarleonard Resolved